api/v1alpha1/selfnoderemediationconfig_types.go

```diff
@@ -127,6 +128,11 @@ type SelfNodeRemediationConfigSpec struct {
 	// CustomDsTolerations allows to add custom tolerations snr agents that are running on the ds in order to support remediation for different types of nodes.
 	// +optional
 	CustomDsTolerations []v1.Toleration `json:"customDsTolerations,omitempty"`
+
+	// +kubebuilder:default:=1
+	// +kubebuilder:validation:Minimum=0
+	// Minimum number of peer workers/control nodes to attempt to contact before deciding if node is unhealthy or not
```

Comment:
I might understand the minimum value for workers, but for CP? Do we want to limit the peers to contact for both workers and CP, even if we have more than one?

Reply:
We do support single node cluster install for some configs that go out the door

Reply:
@clobrano checking to see if this can be resolved based on the latest changes.  Thanks!

Reply:
Yes, after updating CRD files as Marc asked, lgtm